### ISSUE TITLE
islandmanager Duplicate Key causes plugin to be disabled

### ISSUE THREAD

**Issue by youwebaba** [IridiumLLC/IridiumSkyblock#242]
[islandmanager.zip](https://github.com/IridiumLLC/IridiumSkyblock/files/5147364/islandmanager.zip)
Duplicate Key seems to happen randomly, even after **manually** removing the duplicate key, it will reappear after some time and cause the plugin to be disabled.
Running Paper-150.jar as server file on 4 GB of RAM.
if you need anything else, please let me know. Thank you

**Comment by MertUnverdi** [comment]
The source and cause of this problem are not fully resolved
An error with 2.4.7
Use 2.4.6 and wait for the new version

**Comment by octo-kumo** [comment]
```
[12:09:16 INFO]: [IridiumSkyblock] Enabling IridiumSkyblock v2.4.6
[12:09:16 INFO]: [IridiumSkyblock] Unsupported Version Detected: v1_16_R2
[12:09:16 INFO]: [IridiumSkyblock] Try updating from spigot
[12:09:16 INFO]: [IridiumSkyblock] Disabling IridiumSkyblock v2.4.6
[12:09:16 INFO]: [IridiumSkyblock] -------------------------------
[12:09:16 INFO]: [IridiumSkyblock] 
[12:09:16 INFO]: [IridiumSkyblock] IridiumSkyblock Disabled!
[12:09:16 INFO]: [IridiumSkyblock] 
[12:09:16 INFO]: [IridiumSkyblock] -------------------------------
```
The above error is thrown when running 2.4.6 on 1.16.2

**Comment by MertUnverdi** [comment]
2.4.6 unsupports for 1.16.2 you can use with 1.16.1
Or wait to 2.4.8

**Comment by IridiumLLC** [comment]
Should be fixed in 2.4.8
